django_netjsongraph/tests/test_visualizer.py

```diff
@@ -31,5 +32,5 @@ def test_detail_uuid_exception(self):
         """
         see https://github.com/interop-dev/django-netjsongraph/issues/4
         """
-        response = self.client.get('/topology/a083b494-8e16-4054-9537-fb9eba914861-wrong/')
+        response = get_topology_or_404('/topology/a083b494-8e16-4054-9537-fb9eba914861-wrong/')
```

Comment:
`self.client.get` simulates an HTTP request to the view, see the [django docs regarding the test client](https://docs.djangoproject.com/en/1.11/topics/testing/tools/)